### ISSUE TITLE
Usage sample

### ISSUE THREAD

**Issue by xperiandri** [demetrixbio/Plough.WebApi#6]
Is there a sample project to have a look at?

**Comment by daz10000** [comment]
Unfortunately no ;( -   what would be most helpful.  I have meant to do this "properly" for a while but maybe the best good would be hello world,  or a todo app or something.  I really wanted to build something useful that would also demo it.   How important is a working auth, or database interaction for example.  One option might be to build a demo on of of the `dvdrental` database we use in the `FSharp.Npgsql` project - but who needs a dvd library rental app.

**Comment by daz10000** [comment]
p.s. if writing a simple version of one of those was interesting for you,   we can probably get on chat,  get you started and if you wanted to send in a simple app once we talk through it,  that might be a great addition to the repo.  We'd glad accept it.

**Comment by xperiandri** [comment]
Yes, that is a good idea. Do you mean scheduling a 30 min maximum call to get started together?

**Comment by daz10000** [comment]
Whatever would work best / zoom call or q/a on a chat client.   I will see if I can find time to at least push the bare minimum example out to the repo,  we owe you that at least ;)
